integration/test_dynamic_routes.py

```diff
@@ -69,7 +69,7 @@ def redirect_page():
 
 @pytest.fixture(scope="session")
 def dynamic_route(
-    app_harness_env: Type[AppHarness], tmp_path_factory
+        app_harness_env: Type[AppHarness], tmp_path_factory
```

Comment:
i don't understand why these blocks are all indented more...

Reply:
probably as a result of running precommits (black), not typically sure why those parts are affected